content/posts/chapelcon25-retro/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Reflections on ChapelCon '25"
+date: 2025-11-05
+tags: ["ChapelCon", "Community"]
+series: []
+summary: "This post is a retrospective on ChapelCon '25"
+weight: 15
+authors: ["Brandon Neth"]
+---
+
+ChapelCon '25 is behind us, and another year of productive HPC programming lies ahead!
+Don't worry if you missed a talk, you can find the recordings and slides on the ChapelCon '25 [webpage](https://chapel-lang.org/chapelcon25/).
+Before reflecting on the conference, I'd like to thank everyone who made ChapelCon '25 the exciting week it was. 
+Thank you to contributors and participants for the exciting dialogue during our demo sessions and conference days.
+Thank you to the program committee for your support in reviewing a record-breaking number of submissions.
+And, of course, huge thank yous to Luca Ferranti for his work as Program Committee Chair, Daniel Fedorin for his as Tutorial Day Chair, and Jade Abraham for theirs as Office Hours Chair.
+
+### Second Conference, First Experiments
+
+As the second instance of ChapelCon, this year's conference was our first opportunity to experiment with some changes to the conference format.
+We tried out four changes to the conference: a season change from early summer to early autumn, an expansion from three days to four, a new format for tutorial days, and a new approach to the submission and review process.
+All four changes were largely successful, both in terms of improving the conference experience and teaching important lessons about what makes ChapelCon a valuable community gathering. I'd like to offer some reflections on a couple of them.
+
+One of the first program decisions we made was to expand the conference from 3 days to 4.
+This stemmed from two observations. 
+First, last year's attendees were extremely enthusiastic about last year's tutorial and coding days.
```

Comment:
Maybe drop replace the second "last year's" with "the" or "the introduction of" to avoid repeating "last year's" twice in rapid succession.